include/awkward/builder/Builder.h

```diff
@@ -78,6 +78,14 @@ namespace awkward {
     virtual const BuilderPtr
       complex(std::complex<double> x) = 0;
 
+    /// @brief Adds a datetime64 value `x` to the accumulated data.
+    virtual const BuilderPtr
+      datetime64(int64_t x, const std::string& unit) = 0;
+
+    /// @brief Adds a timedelta64 value `x` to the accumulated data.
+    virtual const BuilderPtr
+      timedelta64(int64_t x, const std::string& unit) = 0;
+
```

Comment:
For consistency, drop the "64" in these method names...

Reply:
done